imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,82 @@
+from django.test import TestCase
+import factory
+from . import models
+from .models import Photos, Album
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = models.User
+        django_get_or_create = ('username',)
+    username = 'people'
+
+
+class PhotoFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Photos
+
+    user = UserFactory.create(username='user1')
+
+
+class AlbumFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Album
+
+    user = UserFactory.create(username='user2')
+
+
+class TestPhoto(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.user1_photo = PhotoFactory.create()
+        cls.user1_photo = PhotoFactory.create()
+
+    def test_photo_belongs_to_unique_user(self):
+        assert self.user1_photo.user.username == 'user1'
+
+    def test_new_empty_description(self):
+        assert self.user1_photo.description == ''
+
+    def test_new_empty_title(self):
+        assert self.user1_photo.title == ''
+
+    def test_photo_access(self):
+        assert self.user1_photo.published == 'private'
+        self.user1_photo.published = 'public'
+        assert self.user1_photo.published == 'public'
```

Comment:
Because you are using `setUpClass` above, your change to the published state of this photo will be present in any tests that happen after this one.  At the moment, that isn't any, since there are no tests alphabetically later than this one, but that will not always be the case.